CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 0.14.0-rc1 / 2015-05-21
```

Comment:
As discussed, let's only add a changelog entry for the full 0.14.0 release (non-RC).